CMakeLists.txt

```diff
@@ -12,19 +12,21 @@ endif ()
 
 set (CMAKE_C_STANDARD 99)
 
-option (ENABLE_SHARED_BSON "Dynamically link libbson (default is static)" OFF)
```

Comment:
This option was moved to `ImportBSON.cmake`, and renamed to `USE_SHARED_LIBBSON`.